installer/templates/phx_single/mix.exs

```diff
@@ -70,7 +70,12 @@ defmodule <%= @app_module %>.MixProject do
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
       test: ["ecto.create --quiet", "ecto.migrate --quiet", "test"]<% end %><%= if @assets do %>,
-      "assets.setup": ["tailwind.install --if-missing", "esbuild.install --if-missing"],
+      "assets.setup": [
+        "tailwind.install --if-missing",
+        "esbuild.install --if-missing",
+        "tailwind default",
+        "esbuild default"
```

Comment:
Does the order matter? If it doesn't my gut tells me esbuild should run first as it seems lower level. lmk!

```suggestion
        "esbuild.install --if-missing",
        "tailwind.install --if-missing",
        "esbuild default",
        "tailwind default"
```